application/common/ipc/electron.ipc.messages/file.check.request.ts

```diff
@@ -0,0 +1,30 @@
+export interface IFileCheckRequest {
+    guid: string;
+    files: string[];
+}
+
+export class FileCheckRequest {
+    public static signature: string = 'FileCheckRequest';
+    public signature: string = FileCheckRequest.signature;
```

Comment:
Probably `FileCheckRequest` isn't a self-explaining name. Actually we do not check files, but we get a "flat" list of files from files/folders. Please use some other a self-explaining name of IPC message (and IPC file)